src/current/v22.2/developer-basics.md

```diff
@@ -16,15 +16,19 @@ Note that the sections that follow were written for the purposes of orienting ap
 
 ### How transactions work in CockroachDB
 
-CockroachDB is designed to make your data scalable and consistent. All stored records are replicated across a distributed deployment of database instances, and all database transactions committed in CockroachDB satisfy [ACID properties](https://en.wikipedia.org/wiki/ACID) and the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem). This means that when you store data in CockroachDB, the data will be valid and anomaly-free, even in the event of a system error or power failure.
+CockroachDB is designed to make your data scalable and consistent. All stored records are replicated across a distributed deployment of database instances, and all database transactions committed in CockroachDB satisfy [ACID properties](https://en.wikipedia.org/wiki/ACID). This makes CockroachDB a CAP-Consistent (CP) system under the [CAP theorem](https://wikipedia.org/wiki/CAP_theorem), and means that when you store data in CockroachDB, the data will be valid and anomaly-free, even in the event of a system error or power failure.
```

Comment:
There is unfortunately no connection between the C in ACID and the C in CAP, so saying "this makes" is not correct. I suggest removing any mention of the CAP theorem from this paragraph and putting the new CAP-related paragaphs in a new section "How replication works in CockroachDB" (or omitting them? ACID transactions belong in a "developer basics" doc but I'm not sure the CAP theorem does)

Reply:
We don't really talk about the CAP theorem in the architecture docs. Is that a better place for this info?